_toc.yml

```diff
@@ -30,27 +31,32 @@ parts:
       - glob: Satellite_ECVs/Ocean/*
 
 - caption: ➡️ Insitu Observations
+  numbered: 2
   chapters:
   - file: In_Situ/insitu.md
     title: Quality assessments for Insitu Observations
     sections:
     - glob: In_Situ/*
 
 - caption: ➡️ Reanalysis
+  numbered: 2
   chapters:
   - file: Reanalyses/reanalysis.md
     title: Quality assessments for Reanalysis
     sections:
     - glob: Reanalyses/*
 
 - caption: ➡️ Seasonal Forecasts
+  numbered: 2
   chapters:
   - file: Seasonal_Forecasts/seasonal.md
     title: Quality assessments for Seasonal Forecasts
     sections:
-    - glob: Seasonal_Forecasts/*
+    - file: Seasonal_Forecasts/seasonal_seasonal-monthly-single-levels_forecast-skill_q02.ipynb
```

Comment:
why does the glob need to be removed here?

Reply:
sorry my bad, I was trying various things and I forgot to restore the original